src/components/LeverageSlider/LeverageSlider.tsx

```diff
@@ -127,3 +128,40 @@ function generateKeyValueMap(marks: number[]) {
 
   return { keyValueMap, valueKeyMap };
 }
+
+function lerpValue2SliderKey(keyValueMap: { [key: number]: number }, value: number): number {
```

Comment:
lerp confuses here, it in fact doesn't matter what kind of value is passed here, single value can't be "lerped"

I'd call it valueToSliderKey / sliderValueToSliderKey

Reply:
done